### ISSUE TITLE
Please block the yandex.com search engine, because playing porn videos and pictures is possible directly in the search engine

### ISSUE THREAD

**Issue by jobbb2** [Sinfonietta/hostfiles#292]
Please block yandex.com (search-engine),
For example if you type in your browser: "yandex.com" and type in the word "porn" and then click on tab video, you can show porn-videos as a preview directly in the yandex search-engine in full lenght.
Also if you click on the tab "images" you can show thousends of porn-pictures directly in the yandex search-engine.
With yandex.com you will find a huge selection of porn, much much more than other search engines.

I created this thread in case anyone asks about unblocking this search engine. So that it is clear why the search engine was blocked.

**Comment by thomasmerz** [comment]
Any other search engine (google, bing, …) could also show you this kind of…

❓ So why only add one specific search engine - and not all (which would really make no sense)? 😉 

ℹ️  Maybe you should try these hacks for google, bing and youtube with your local hosts-file (or pi-hole, adguard home, … for your whole network at home):
https://support.google.com/websearch/answer/186669#zippy=%2Cif-you-manage-a-mac-windows-or-linux-device%2Con-linux
https://support.opendns.com/hc/en-us/articles/227986807-How-to-Enforcing-Google-SafeSearch-YouTube-and-Bing

Google and Youtube work very well, but Bing Restricted is easy to bypass (images and videos are safe, but you get all the links to these sites and you can change level of safesearch or even deactivate it with some clicks!)

@Sinfonietta - should we close this as "won't be worked on"?

**Comment by jobbb2** [comment]
Hello @thomasmerz,
please test it not in your mind, please test it by doing. completely neutral. And then you will find out, that this is not possible in other search-engines. Otherwise please send me an evidence-link.
The pornos are integrated in the search-engine, so its not possible to block the porn-links in a filterlist. For example if you block the porno link with a filterlist.. .its not working... Nevertheless, you can see always the porno-video within the yandex-search-engine!!

@Sinfonietta - we should add it to the filterlist.

**Comment by leongjs98** [comment]
> Also if you click on the tab "images" you can show thousends of porn-pictures directly in the yandex search-engine.
With yandex.com you will find a huge selection of porn, much much more than other search engines.

If you turn off safe search in Duckduckgo and Google, you can still view porn images and porn gifs.
I don't think it make sense to add a search engine

**Comment by thomasmerz** [comment]
@Sinfonietta , this issue should be closed as "wontfix", shouldn't it? 🤷🏼‍♂️
